### ISSUE TITLE
Graduation Project

### ISSUE THREAD

**Issue by Mr-DOW** [Mr-DOW/Graduation-Project#1]
Project Title: Design and Implementation of Defogging Method Based on Dark Channel Prior
 Project Introduction and Results Requirements: As science and technology become more advanced, multimedia image processing technology has now been more and more widely applied to various fields. However, there is a serious limitation of this technique, that is, the source of the image is affected by various weather factors such as foggy weather, rainy and snowy weather, hazy dust and the like, so that the image obtained by the camera is subjected to larger or smaller The degree of weakening and blurring, leading to the edge of the image, especially the scene becomes blurred, the loss of the original high contrast, in serious cases will cause the image color information distortion or offset. In order to be able to work in all areas of the surveillance system under all sorts of harsh and extreme weather conditions, we need to make these degraded images image intensified and made clear. In recent years, a fast and effective physical model-based defogging method based on a priori assumption has attracted the attention of many scholars, which is based on the priori method of Dr. He Kai-ming, using the color channel method, using the nature The inherent model and law as the prerequisite to restore the image, can be more concise and clear enough. The subject studies the physical processes of image degradation and foggy image fog mechanism. The defogging method based on the dark channel a priori principle is studied, and the effectiveness of this method is illustrated through experimental simulation. On the basis of this, the relationship between fog concentration and image saturation and brightness in different color spaces is explored, and the corresponding improvement methods are designed. Finally, the effectiveness of the defogging method based on the dark channel a priori principle and the effect of the improved defogging method based on the dark channel a priori principle are demonstrated experimentally. Subject needs to be completed with the basic knowledge of image analysis. Experimental platform for Python or Matlab development platform.
The main equipment needed for the project: Raw conditions and data 1. Training data set for de-fog method based on dark channel a priori principle 2. Sample of de-fog method based on dark channel a priori principle for reference Examples; 3. Python or Matlab related books and materials; technical requirements need to be familiar with the basic theory of image processing and image de-fog related algorithms. Need to master the programming work associated with Python or Matlab. The graduation design requirements: First, the subject needs to complete the basic knowledge of image processing. Second, the completion of undergraduate completion of the task for the understanding of the correct; Third, the need to complete the completion of those who have a basic knowledge of Python or Matlab; Fourth, required to be able to achieve the corresponding simulation program can be more detailed to provide the corresponding Program manual
